src/components/editor/GroupChoiceModal.jsx

```diff
@@ -28,22 +24,16 @@ const GroupChoiceModal = (props) => {
   }
 
   const saveAndClose = () => {
-    props.saveStarted()
-    const request = publishRDFResource(props.currentUser, props.rdf(), selectedValue)
-
-    request.then((result) => {
-      props.setBaseURL(result.response.headers.location)
-      // Need to regenerate RDF now that have baseURL
-      props.saveFinished(props.rdf())
-    }).catch((err) => {
-      alert('Unable to save resource')
-      console.error('unable to save resource')
-      console.error(err)
-    })
+    props.publishMyResource(props.currentUser, selectedValue)
     props.closeRdfPreview()
     props.close()
   }
 
+  if (props.error) {
+    // the error will be displayed via Editor -> ErrorMessages components
+    return (<div />)
```

Comment:
I don't think that saveAndClose is expected to return any values ever.

Reply:
it doesn't, now.  Am i missing something?

Reply:
I've removed these useless lines, which weren't actually in saveAndClose() in any case.